lineflow/datasets/wikitext.py

```diff
@@ -0,0 +1,20 @@
+from lineflow.download import cached_download
+from lineflow.text import TextDataset
+
+
+TRAIN_URL = 'https://raw.githubusercontent.com/sobamchan/wikitext-2/master/train.txt'
+VALID_URL = 'https://raw.githubusercontent.com/sobamchan/wikitext-2/master/valid.txt'
+TEST_URL = 'https://raw.githubusercontent.com/sobamchan/wikitext-2/master/test.txt'
+
+
+class WikiText(TextDataset):
```

Comment:
Could you inherit `lineflow.Dataset` rahter than `lineflow.TextDdataset`.
Sorry for the inconvenience. These days, I've changed the internal structure of this project.
I'd like you to follow this way: [lineflow.datasets.Squad](https://github.com/yasufumy/lineflow/blob/master/lineflow/datasets/squad.py#L15)
Briefly, please inherit `lineflow.Dataset` and use `lineflow.text.RandomAccessText`.
Thank you.

Reply:
I think we should name this class `WikiText2` to avoid ambiguity.
There is the WikiText-103 dataset.